public/checkout.js

```diff
@@ -130,21 +124,17 @@ var packeteryLoadCheckout = function( $, settings ) {
 				destinationAddress.city.val(getRateAttrValue(carrierRateId, 'packetery_address_city', ''));
 				destinationAddress.postCode.val(getRateAttrValue(carrierRateId, 'packetery_address_postCode', ''));
 
-				$widgetDiv.find('.packeta-widget-selected-address').html(
-					getRateAttrValue(carrierRateId, 'packetery_address_street', '')
-					+ ' ' +
-					getRateAttrValue(carrierRateId, 'packetery_address_houseNumber', '')
-					+ ', ' +
-					getRateAttrValue(carrierRateId, 'packetery_address_city', '')
-					+ ', ' +
-					getRateAttrValue(carrierRateId, 'packetery_address_postCode', '')
-					+ ' ' +
+				var $estimatedDeliveryDateSection = $('.estimated-delivery-date');
+
+				if ( typeof getRateAttrValue( carrierRateId, 'packetery_car_delivery_id', false ) !== 'undefined' ) {
```

Comment:
Není to zbytečná podmínka? Nekontroluje se typ vybrané přepravní metody už před voláním showCarDeliveryAddress?

Reply:
Neni to vybrany typ prepravni metody, ale id primo te car delivery, co prijde z widgetu, ale mas pravdu, par radku nad tim uz to resim, takze odstranuji.